lib/ffmpeg/reporters/progress.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require_relative 'output'
+
+module FFMPEG
+  module Reporters
+    # Represents the progress of an encoding operation.
+    class Progress < Output
+      def self.match?(line)
+        line =~ /^\s*frame=/ ? true : false
```

Comment:
if we don't need the results of the regex, then `line.match?(/.../)` should be used to get a boolean value

Reply:
done